src/components/Header/MobileMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import SearchInput from './SearchInput';
+
+import CrossSVG from '../../assets/svg/cross.svg';
+import { IMobileMenuProps } from './Types';
+import AboutList from './AboutList';
+import StoreList from './StoreList';
+import { useState } from 'react';
+
+const MobileMenu: React.FC<IMobileMenuProps> = ({ close }) => {
+  const [storeListOpen, setStoreListOpen] = useState<boolean>(false);
+  const [aboutListOpen, setAboutListOpen] = useState<boolean>(false);
+
+  return (
+    <div className="mobile-menu">
+      <div className="container">
+        <div
+          className="mobile-menu__cross-icon"
+          role="presentation"
+          onClick={close}
+        >
+          <CrossSVG />
+        </div>
+        <SearchInput />
+        <p role="presentation" onClick={() => setStoreListOpen(!storeListOpen)}>
+          Магазин
+        </p>
+        {storeListOpen && (
```

Comment:
use css to hide a component and add animation when about and shop switch like on bevza.com